main.js

```diff
@@ -1,10 +1,15 @@
 var app = require('app')
 var BrowserWindow = require('browser-window')
 var crashReporter = require('crash-reporter')
+var Menu = require('menu')
 
-crashReporter.start()
+var darwinTemplate = require('./darwin-menu.js')
```

Comment:
When we `require` this file and set it to the variable `darwinTempalte` it's the same as:

```js
var darwinTemplate = function menu(app, mainWindow) { .... }
```

So when we use it below :arrow_down: